test/threads.jl

```diff
@@ -200,7 +202,14 @@ close(proc.in)
             done[] = true
             close(timer)
         end
-        @test success(proc)
+        if Sys.iswindows() && (n >= 2000000)
+            # Known failure https://github.com/JuliaLang/julia/issues/43124
+            @test_skip success(proc)
```

Comment:
Please feel free to decrease `n` if more failures are found.

Reply:
Sounds good!